### DISCUSSION TITLE
Add map submitted/ranked info in song select

### DISCUSSION THREAD

**Discussion by kwotaq** [ppy/osu#17390] [Ideas] [answered]
I've noticed while in song select you can go to the details option of a beatmap and see basically everything you need to know EXCEPT when it was submitted and ranked. I suggest adding that info as well as its quite tiresome having to search the map up in direct every time you want to know that info

**Comment by bdach** [comment, marked as answer]
planned to be added [in the upcoming redesign](https://www.figma.com/file/DXKwqZhD5yyb1igc3mKo1P/Song-Select-Draft-2?node-id=0%3A1)
